src/compose/ComposeBox.js

```diff
@@ -265,13 +266,21 @@ class ComposeBox extends PureComponent<Props, State> {
         showErrorAlert(error.message, 'Failed to edit message');
       });
     }
-
     onEditComplete();
     if (this.messageInput) {
       this.messageInput.blur();
     }
   };
 
+  cancelEdit = () => {
+    const { onEditComplete } = this.props;
+
+    if (this.messageInput) {
+      this.messageInput.blur();
+    }
+    onEditComplete();
```

Comment:
These come in the opposite order from above in `handleEdit`.

Probably both are fine :slightly_smiling_face: but I'd rather not have to think about both -- better to pick one.